### DISCUSSION TITLE
Welcome to bot-topia Discussions!

### DISCUSSION THREAD

**Discussion by LofiMonk** [LofiMonk/bot-topia#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by TH4352** [comment]
I love this mod!
